ios/RNSScreenStackHeaderConfig.m

```diff
@@ -138,10 +138,21 @@ - (void)updateViewControllerIfNeeded
   }
 }
 
+- (void)layoutNavigtationController
+{
+  UIViewController *vc = _screenView.controller;
+  UINavigationController *navctr = (UINavigationController *)vc.parentViewController;
+  [navctr.view setNeedsLayout];
+  [navctr.view layoutIfNeeded];
+}
+
 - (void)didSetProps:(NSArray<NSString *> *)changedProps
 {
   [super didSetProps:changedProps];
   [self updateViewControllerIfNeeded];
+  if ([changedProps containsObject:@"translucent"]) {
```

Comment:
This feels kinda hacky and it's wishful thinking that we won't add anything new with "translucent" in the name to unnecessarily redraw the hierarchy.